src/components/ui/rootHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from "next/image";
+import React from "react";
+
+import { CircleUser } from "lucide-react";
+import { RoutePath } from "@/lib/route-path";
+import { getLoggedInSession } from "@/lib/auth";
+import Link from "next/link";
+
+const RootHeader = async () => {
+  const session = await getLoggedInSession();
+  const accountType = session?.accountType;
+
+  return (
+    <nav className="sticky top-0 z-10 bg-white">
+      <div className=" flex items-center justify-between border-b shadow-lg">
```

Comment:
Need to make this 72px high.

<img width="470" alt="image" src="https://github.com/barkbank/barkbank-pawtal/assets/2757078/3ba51b01-edb7-4bc4-b12b-ee0239f660da">

Reply:
Done [6c90f25](https://github.com/barkbank/barkbank-pawtal/pull/61/commits/6c90f25d287abf8373fb9862fd2eccaaf3a771fe)